contest/templates/contest/contestArchive.html

```diff
@@ -52,61 +54,71 @@
     <thead>
       <tr>
         <th>
-          {% if admin %}
-          <button type="button" class="btn btn-success btn-xs" data-toggle="modal" data-target="#contestInfo">
-            <span class="glyphicon glyphicon-plus"></span>
-          </button>
+          {% if user|has_judge_auth %}
+          <a href="{% url 'contest:new' %}">
+            <button type="button" class="btn btn-success btn-xs" >
+              <span class="glyphicon glyphicon-plus"></span>
+            </button>
+          </a>
           {% endif %}
         </th>
         <th>#</th>
         <th>Contest</th>
         <th>Start time</th>
         <th>End time</th>
-        <th>Contestant</th>
         <th>Owner</th>
       </tr>
     </thead>
     <tbody>
-      {% for contest in contest_list %}
+      {% for contest in contests %}
```

Comment:
Please not show contests whose start time is later than current time for normal users.